scanomatic/ui_server_data/fixture.html

```diff
@@ -84,28 +79,28 @@ <h2 id="selected-fixture-name"></h2>
       <input type="hidden" name="update-type" id='update-type' value="update"/>
       <div>
         <button type="button" id="clear-areas"
-            onclick="clearAreas();draw_fixture();selected_fixture_canvas_jq.focus();">
+            onclick="som.clearAreas();som.drawFixture();selectedFixtureCanvasJq.focus();">
```

Comment:
```suggestion
            onclick="som.clearAreas();som.drawFixture();som.selectedFixtureCanvasJq.focus();">
```

Reply:
?

Reply:
Hmm... this is a global variable defined in `fixture.js`:
```javascript
let selectedFixtureCanvasJq;
```
I'll have to figure out how to handle that... might be a few (or a bunch) more that I have missed...

Reply:
I think this is fixed now.

Reply:
looks reasonable here now